database/insertData/laos/03_template_license_MMC_MMD.js

```diff
@@ -510,56 +498,169 @@ exports.queries = [
+                            elementTypePluginCode: "radioChoice"
+                            category: QUESTION
+                            parameters: {
+                              label: "Type of role"
+                              options: ["Government", "Private Sector"]
+                              layout: "inline"
+                            }
+                          }
+                          {
+                            code: "PEname"
+                            title: "Org name"
+                            elementTypePluginCode: "shortText"
+                            category: QUESTION
+                            parameters: {
+                              label: "Name of institution or company"
+                              maxLength: 120
+                            }
+                          }
+                          {
+                            code: "PEorgTel"
+                            title: "Org telephone"
+                            elementTypePluginCode: "shortText"
+                            category: QUESTION
+                            isRequired: false
+                            parameters: {
+                              label: "Telephone"
+                              maxLength: 120
+                            }
+                            validation: {
+                              operator: "REGEX"
+                              children: [
+                                {
+                                  operator: "objectProperties"
+                                  children: ["responses.thisResponse"]
+                                }
+                                "^[0-9()\\\\-\+]*$"
+                              ]
+                            }
+                            validationMessage: "Not a valid phone number"
+                          }
+                          {
+                            code: "PEorgEmail"
+                            title: "Org email"
+                            elementTypePluginCode: "shortText"
+                            category: QUESTION
+                            isRequired: false
+                            parameters: {
+                              label: "Email"
+                              maxLength: 120
+                            }
+                            validation: {
+                              operator: "REGEX"
+                              children: [
+                                {
+                                  operator: "objectProperties"
+                                  children: ["responses.thisResponse"]
+                                }
+                                {
+                                  value: "^[A-Za-z0-9.]+@[A-Za-z0-9]+\\\\.[A-Za-z0-9.]+$"
+                                }
+                              ]
+                            }
+                            validationMessage: "Not a valid email address"
```

Comment:
Same thing about re-uploading documents on this one: 
- **LMMD02** – CV of applicant with recent (max 1 year) photo
- **LMMD06** – Certificate of education level
- **LMMD08** – Letter from previous employers documenting at least 3 years professional experience (for private sector must be certified by provincial or capital level)
- **LMMD09** – Documentation showing resignation from previous employment (for private sector must be certified by provincial or capital level)
- **LMMD09** – Documentation showing resignation from previous employment (for private sector must be certified by provincial or capital level)